src/imagej/util.py

```diff
@@ -0,0 +1,105 @@
+from itertools import cycle
+from shutil import get_terminal_size
+from threading import Thread
+from time import sleep
+from typing import List
+
+
+class Loader:
+    def __init__(
+        self, start_msg="Loading...", end_msg="Done!", timeout=0.1, style="rotate"
```

Comment:
Can you think of a good situation where you'd want `end_msg` opposed to `end_suffix`?

In your use cases, the only difference between `start_msg` and `end_msg` is the "Done!". You could simplify the calls if you only have to pass the on-completion suffix.

Reply:
Agreed! (742427bcc6d45e521e39829026e8cf7d468bfb98).